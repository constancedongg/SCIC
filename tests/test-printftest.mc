int func main() {
    printf(1.2);
<<<<<<< HEAD
=======
    printf(0.8181818);
    printf(1.2345678);
    printf(103.1243);
    return 0;
>>>>>>> main
}