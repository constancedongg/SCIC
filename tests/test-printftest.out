1.2
0.818182
1.23457
103.124